File: clients/java/src/main/java/net/weweave/tubewarder/client/Address.java
package net.weweave.tubewarder.client;

import java.io.Serializable;

public class Address implements Serializable {
    private String address;
    private String name;

    public Address() {

    }

    public Address(String address) {
        setAddress(address);
    }

    public Address(String address, String name) {
        setAddress(address);
        setName(name);;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: clients/php/Address.php
<?php
namespace Tubewarder;

class Address {
    private $address;
    private $name;

    public function __construct($address = '', $name = '') {
        $this->address = $address;
        $this->name = $name;
    }

    /**
     * @return string
     */
    public function getAddress() {
        return $this->address;
    }

    /**
     * @param string $address
     */
    public function setAddress($address) {
        $this->address = $address;
    }

    /**
     * @return string
     */
    public function getName() {
        return $this->name;
    }

    /**
     * @param string $name
     */
    public function setName($name) {
        $this->name = $name;
    }

    /**
     * @return array
     */
    public function getObject() {
        $o = array(
            'address' => $this->getAddress(),
            'name' => $this->getName()
        );
        return $o;
    }
}